src/components/SplitButton/SplitButtonMenu/__tests__/splitButtonMenu-tests.jest.tsx

```diff
@@ -39,7 +39,7 @@ describe("SplitButtonMenu", () => {
 
   it("should focus first menu item on mount", () => {
     const { getByTestId } = renderSplitButtonMenu();
-    const firstMenuItemId = `${getTestId(ComponentDefaultTestId.MENU_ITEM)}0`;
+    const firstMenuItemId = `${getTestId(ComponentDefaultTestId.MENU_ITEM)}_0`;
```

Comment:
i didn't really get  the change here with the data test ids let's talk about it?